### ISSUE TITLE
Why there isn't a sub button, Command

### ISSUE THREAD

**Issue by Frozenrunner159** [natzert/PrintMakingBot#21]
"Our main intention being here on Twitch is to let people see the printmaking, and to generally spread word about our work, not for funding. So at present, we haven't 'turned on' the affiliate or partner options that are available to us here" - Dave

**Comment by jstoxrocky** [comment]
Yeah, great idea. We should start a general issue thread of questions Dave get's asked a lot. To be used for writing bot commands at a later date. His right hand tool used as a lefty, for example.

**Comment by jstoxrocky** [comment]
"Dave, what is your favourite...."

**Comment by Frozenrunner159** [comment]
"Are you in tokyo?" "do you speak Japanese?" "How long have you been In japan"

**Comment by jstoxrocky** [comment]
Maybe even a company directory, I know he is forever explaining who Suga-san is etc. and I can never keep it straight in my mind.

**Comment by Frozenrunner159** [comment]
"it is kawai-san not kawaii-san" But yea like the !cameron command

**Comment by jstoxrocky** [comment]
Out of curiosity, how did the short-lived "Did you take the paper out?" bot thing run? I never saw a PR for that. Seems like it lives somewhere else. Would have been helpful to have more eyes on that code before it went nutzo.
